### ISSUE TITLE
E2E Keyboard Test Port to Cypress

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#15784]
## Problem Statement 
*Keyboard E2E tests are not currently being done, because the pilot teams have switched over to Cypress, which does not have that capability currently. How might we enable teams to conduct keyboard navigation-style E2E tests*

## Hypothesis or Bet
*By porting the previous tests over from Nightwatch into Cypress, we'll allow teams to begin conducting Keyboard E2E tests again*

## Objectives
*O3: Facilitate VFS and VSP teams meeting or improving DEPO and VA.gov standards for security, consistency, and quality.*

## We will know we're done when... ("Definition of Done")
*All previous helper methods have been ported over to Cypress and work as intended, and have created helper methods for accordions/hidden content.*

## Known Blockers/Dependencies 
*_Possible Dependency: Cypress doesn't register space/return key entries currently_*

## Projected Launch Date
*_3 Sprints - Mid/Late-December_*

### ... tested?
- [ ] Write tests that integrate with existing Cypress form test helpers: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/tests/feedback-tool/01.feedback-tool-keyboard.e2e.spec.js
- **NOTE:** Tests should not rely on Generated ID's for CSS selectors

### ... documented?
- [ ] Clear API for each helper method
- [ ] Should introduce what we're trying to accomplish and why Keyboard tests are important in the E2E pipeline
- [ ] Who to contact if there are issues (Trevor & Noah)

### Documentation _(IN PROGRESS)_
* **`PRODUCT_NAME`**: _directory name used for your product documentation_
* **Product Outline**: [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/cypress_e2e_testing/cypress_keyboard_test_product-outline.md)

* **README**: _link to VFS-facing README for your product_
* **User Guide**: _link to User Guide_

### Measurement (Success Metrics)
- [ ] VFS Teams are using the Keyboard Tests, based on a search for the user of the helpers
- [ ] Tests are not breaking, based number of support requests received

## TODOs
- [ ] Facilitate a conversation between Eugene Doan & Testing Team

## Additional Reference
- [Automated Accessibility/508 Compliance Testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/testing/508-automated-testing.md)
- [Future Accessibility Tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#future-accessibility-tests)

**Comment by pjhill** [comment]
This is complete.
